server/utils/personUtils.ts

```diff
@@ -9,7 +9,7 @@ const tierBadge = (tier: string): string => {
 }
 
 const isApplicableTier = (sex: string, tier: string): boolean => {
-  const applicableTiersAll = ['A3', 'A2', 'B1', 'B3', 'B2', 'B1']
+  const applicableTiersAll = ['A1', 'A3', 'A2', 'B1', 'B3', 'B2']
```

Comment:
amazing that we hadn't noticed that B1 was repeated! 🤦

Reply:
Good idea Ed

Reply:
The jumbled order of the tiers in the array made the duplicate hard to spot so I have put them in order now